src/lang/es.json

```diff
@@ -1,46 +1,75 @@
-{
-  "I18N.LANGUAGE": "Español",
-  "I18N.MAINTAINERS": "@Viriato139ac#0342",
-
-  "WEATHERMANAGE.Title": "Administración de Efectos de Clima",
-  "FILTERMANAGE.Title": "Administración de Efectos de Filtros",
-  "CONTROLS.Effects": "Controles de Efectos",
-  "CONTROLS.Weather": "Controles de Clima",
-  "CONTROLS.Daylight": "Filtro Luz Diurna",
-  "CONTROLS.Underwater": "Filtro submarino",
-  "CONTROLS.Predator": "Filtro de Depredador",
-  "CONTROLS.OldFilm": "Filtro de película antigua",
-  "CONTROLS.Bloom": "Filtro de resplandor",
-  "CONTROLS.SpecialFX": "Efectos Especiales",
-  "CONTROLS.Colorize": "Efectos de Coloración",
-  "CONTROLS.SaveMacro": "Exportar clima como una Macro",
-  "CONTROLS.ClearFX": "Borrar Efectos y Filtros",
-  "EFFECTCONTROLS.Title": "Efectos Especiales",
-  "FXMASTER.AddEffect": "Añadir un Efecto",
-  "FXMASTER.Density": "Densidad",
-  "FXMASTER.Tint": "Tono",
-  "FXMASTER.Scale": "Escalado",
-  "FXMASTER.ScaleHint": "Escalado de Vídeo",
-  "FXMASTER.Speed": "Velocidad",
-  "FXMASTER.SpeedHint": "Velocidad de movimiento del vídeo",
-  "FXMASTER.Direction": "Dirección",
-  "FXMASTER.SaveMacro": "Guardar como Macro",
-  "FXMASTER.SaveChanges": "Guardar Cambios",
-  "FXMASTER.Delete": "Borrar Efectos",
-  "FXMASTER.DeleteConfirm": "<p><strong>¿Estás seguro?</strong> Se borrarán todos los Efectos.</p>",
-  "FXMASTER.Enable": "Habilitar Efectos",
-  "FXMASTER.Angle": "Ángulo",
-  "FXMASTER.AngleHint": "Ángulo inicial del efecto comparado al eje izquierda a derecha",
-  "FXMASTER.Anchor": "Anclaje",
-  "FXMASTER.AnchorHint": "Coordenadas locales del origen entre 0.0 y 1.0",
-  "FXMASTER.File": "Fichero",
-  "FXMASTER.Folder": "Directorio",
-  "FXMASTER.Name": "Nombre",
-  "FXMASTER.AnimationDelay": "Retraso",
-  "FXMASTER.AnimationDelayHint": "Retraso de la animación de velocidad",
-  "FXMASTER.Ease": "Ease",
-  "FXMASTER.EaseHint": "Función de redución de la animación de velocidad",
-  "FXMASTER.PermissionCreate": "Usar efectos especiales FXMaster",
-  "FXMASTER.PermissionCreateHint": "Permitir a los usuarios con el rol seleccionado o superior utilizar los efectos especiales de FXMaster a través de los controles de la escena.",
-  "FXMASTER.MaskWeather": "Enmascarar climatología FXMaster"
-}
+{
+  "I18N.LANGUAGE": "Español",

```

Comment:
Do you know the reason for this and the next line being there? It doesn't seem to be used anywhere, So I suggest to remove these 2 lines unless there is a specific reason for them.